packages/playground/src/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "just-the-docs",
+  "version": "0.2.5",
+  "description": "A modern Jekyll theme for documentation",
+  "repository": "pmarsceill/just-the-docs",
+  "license": "MIT",
+  "bugs": "https://github.com/pmarsceill/just-the-docs/issues",
+  "devDependencies": {
+    "stylelint": "^10.0.1",
+    "stylelint-config-primer": "^7.0.0",
+    "stylelint-selector-no-utility": "^4.0.0",
+    "copy-and-watch": "^0.1.4",
+    "npm-run-all": "^4.1.5",
+    "replace-in-file": "^4.1.1",
+    "rimraf": "^2.6.3",
+    "nps": "^5.9.8",
+    "cross-env": "^6.0.3"
+  },
+  "scripts": {
+    "test": "stylelint '**/*.scss'",
+    "build": "npm-run-all --parallel prepare:** && bundle exec jekyll build --source ./ --destination ../dist",
+    "build:samples": "cd ../../main && nps build.samples",
```

Comment:
instead of building main, we should treat it as a dependency and rely on the root build to build the playground after main.

here we should directly consume the dist folder of main (like main consumes base and does not build it)